pwnlib/dynelf.py

```diff
@@ -641,7 +641,7 @@ def _dynamic_load_dynelf(self, libname):
             p_name = leak.field(cur, LinkMap.l_name)
             name   = leak.s(p_name)
 
-            if libname in name:
+            if libname.encode('utf-8') in name:
```

Comment:
I would prefer `name` get converted to bytes rather than `libname` being encoded as utf-8.  There will likely NEVER be a dynamic library that's not representable by utf-8 chars, but it is likely that we get non-utf8 trash bytes back somehow and explode on decoding instead of doing the always-safe conversion of `name` to `bytes`.